### PR TITLE
Added qualimap rnaseq wrapper.

### PR DESCRIPTION
**Pull request by brcopeland**

### CODE REVIEW

**Comment by brcopeland** [comment]
Just to clarify the change made, the `qualimap` wrapper that runs the java program has a parameter, `--memory-size`, that like you'd expect passes this information onto java.  On the other hand, it does not have any option to modify the java temp directory except through putting it in the environment variable `JAVA_OPTS`, and hence the reason for utilizing that while ignoring `--memory-size`.

File: bio/qualimap/rnaseq/test/Snakefile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+rule qualimap:
+    input:
+        bam = "mapped/a.bam",
+        gtf = "annotation.gtf"
```
Comment (jafors) [inline bio/qualimap/rnaseq/test/Snakefile:4]:
```suggestion
        bam="mapped/a.bam",
        gtf="annotation.gtf"
```
If you format your Snakefiles with `snakefmt`, those things should be easy to catch. ;)

Comment (jafors) [reply bio/qualimap/rnaseq/test/Snakefile:4]: Also, it is always good to add some description for the `input`/`output`/`params` sections in the form of comments in the example Snakefile, even if they seem self-explanatory.

**Review by jafors — CHANGES_REQUESTED** [review]

File: bio/qualimap/rnaseq/test/Snakefile (hunk 1 above)
Comment (brcopeland) [reply bio/qualimap/rnaseq/test/Snakefile]: I've done that now.  I haven't heard of `snakefmt` previously, I'll take a look.

**Review by jafors — APPROVED** [review]
Thanks!

**Pull request merged**